docs/src/pages/components/menus/menus.md

```diff
@@ -33,6 +33,14 @@ To use a selected menu item without impacting the initial focus or the vertical
 
 {{"demo": "pages/components/menus/SimpleListMenu.js"}}
 
+## Positioned menus
```

Comment:
@mbrookes It seems that we are not very consistent in the documentation about the plural. For instance:

- `## Customized tooltips`
- `## Customized accordion`
- `## Positioned Tooltips`
- `## Positioned Popper`
- `### Transition`
- `## Transitions`
- `## Basic alerts`
- `## Basic rating`

I'm not sure what we should do

Reply:
Singular works here. I also wouldn't shy away from switching up the word order where it makes more sense, so: "Menu position"?

Reply:
I guess we could update all the headers at once, that would help maximize consistency.

Reply:
> I guess we could update all the headers at once, that would help maximize consistency.

@oliviertassinari I can singularize the headers for this file as part of this PR if you'd like